examples/pytorch-mnist-membership-inference/demo.ipynb

```diff
@@ -4,14 +4,14 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# PyTorch MNIST Membership Inference demo"
+    "# MNIST Membership Inference demo for Securing AI Lab deployment"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This notebook contains an end-to-end demostration of Dioptra that can be run on any modern laptop."
+    "This notebook contains an end-to-end demostration for the Dioptra architecture that is deployed on the DGX workstation."
```

Comment:
This notebook looks like it is based off of https://github.com/usnistgov/dioptra/blob/main/examples/tensorflow-mnist-classifier/demo-dgx-lab-deployment.ipynb instead of https://github.com/usnistgov/dioptra/blob/29ee50438a001715d1ca0f3038527fe286e74719/examples/pytorch-mnist-membership-inference/demo.ipynb. Generally, we want to try to avoid including instructions about how to connect to specific machines like the DGX in the `demo.ipynb` notebook in each example since most users won't have access to it.

Is it possible to revert the notebook back to https://github.com/usnistgov/dioptra/blob/29ee50438a001715d1ca0f3038527fe286e74719/examples/pytorch-mnist-membership-inference/demo.ipynb and still have the example work? My thought is that the most important changes would potentially be to a couple of the arguments passed to `restapi_client.submit_job()`, such as `entry_point` and `entry_point_kwargs`. If that's the case, revert the notebook and update the arguments as needed.